### PR TITLE
Fixed Physics

### PR DESCRIPTION
**Pull request by CheeseCommando**

### CODE REVIEW

**Review by Khhs167 — CHANGES_REQUESTED** [review]
Please do leave entity AABBs to the entity constructor however. Also please leave _all_ physics in PhysicsBody. Right now the entity also sets it's position to the bounding box which i am not a fan of, the collider should update instead i think. Otherwise it is looking great!

For more clarification, just ask :)

File: game/src/main/java/net/minecraftnt/server/entities/Entity.java [hunk 1]
```diff
@@ -29,8 +36,32 @@ public PhysicsBody addPhysicsBody() {
     public Entity(Vector3 pos){
         this.transform = new Transform(pos);
     }
+    
+    public void setPosition(Vector3 pos) {
+    	this.transform.location = pos;
+    	this.createAABB(boundsX, boundsX);
```
Comment (Khhs167) [inline game/src/main/java/net/minecraftnt/server/entities/Entity.java:42]: boundsX, boundsX should be boundsX, boundsY right?

I just want to ensure it all works properly before merging lol

**Review by Khhs167 — CHANGES_REQUESTED** [review]
i'm okay with this! fix up that minor comment then we're done!

File: game/src/main/java/net/minecraftnt/server/entities/Entity.java (hunk 1 above)
Comment (CheeseCommando) [reply game/src/main/java/net/minecraftnt/server/entities/Entity.java:42]: looks like i missed that lol, but yeah it should be boundsX, boundsY

**Pull request merged**